test/lev-report/charts/HourlyUsage.test.js

```diff
@@ -35,4 +36,21 @@ describe('HourlyUsage component', () => {
 		expect(tree).toMatchSnapshot();
 		success = true;
 	});
+
+	afterAll(() => {
+		if (success && process.argv.includes('--visualise')) {
```

Comment:
Should this be factored out into it's own little module?

That said, I wouldn't spend too much time on it as in future we should provide this sort of functionality through things like Storybook (with Storyshots) and Chromatic.

Reply:
It absolutely could be yes, but as you say I don't think it's currently worth doing - if someone were to add another component before the move to storybook, they could modularise it at that point.

Reply:
Okay.